src/main/java/com/fmi/findmeabuddy/serviceImpl/UserServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package com.fmi.findmeabuddy.serviceImpl;
+
+import com.fmi.findmeabuddy.Service.UserService;
+import com.fmi.findmeabuddy.dao.UserDao;
+import com.fmi.findmeabuddy.model.Account;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class UserServiceImpl implements UserService {
+    @Autowired
+    UserDao userDao;
+
+    @Override
+    public Account findUser(Long accountId) {
+        return userDao.findUser(accountId);
+    }
+
+    @Override
+        public Account loginUser(Long accountId, String password){
+            Account user = this.findUser(accountId);
+            if (user != null & user.getPassword().equals(password)){
```

Comment:
Note that we don't store password in plain text but encode them using BCrypt. You can use this [method](https://docs.spring.io/spring-security/site/docs/3.2.9.RELEASE/apidocs/org/springframework/security/crypto/bcrypt/BCryptPasswordEncoder.html#matches(java.lang.CharSequence,%20java.lang.String)) to check if the password matches.